WeakAurasOptions/OptionsFrames/OptionsFrame.lua

```diff
@@ -518,8 +518,7 @@ function OptionsPrivate.CreateFrame()
   if OptionsPrivate.changelog then
     local changelog
     if OptionsPrivate.changelog.highlightText then
-      changelog = L["Highlights"] .. "\n" .. OptionsPrivate.changelog.highlightText .. "\n"
-      .. L["Commits"] .. "\n" ..OptionsPrivate.changelog.commitText
+      changelog = L["Highlights"] .. "\n" .. OptionsPrivate.changelog.highlightText
```

Comment:
i want to note that because the changelog is a lot more accessible now, we shouldn't be lazy about writing them.

not that we *have* been lazy recently. just sayin'